numcodecs/blosc.pyx

```diff
@@ -228,9 +228,9 @@ def compress(source, char* cname, int clevel, int shuffle=SHUFFLE,
     clevel : int
         Compression level.
     shuffle : int
-        Either NOSHUFFLE (0), SHUFFLE (1), BITSHUFFLE (2) or AUTOSHUFFLE (-1). If -1
-        (default), bit-shuffle will be used for buffers with itemsize 1,
-        and byte-shuffle will be used otherwise.
+        Either NOSHUFFLE (0), SHUFFLE (1), BITSHUFFLE (2) or AUTOSHUFFLE (-1). If -1,
```

Comment:
Maybe worth saying `AUTOSHUFFLE` instead of `-1` for clarity

```suggestion
        Either NOSHUFFLE (0), SHUFFLE (1), BITSHUFFLE (2) or AUTOSHUFFLE (-1). If AUTOSHUFFLE,
```